proposal-membership.md

```diff
@@ -1,22 +1,20 @@
 # Membership
 
-## Active Members
```

Comment:
Removing this line is just to improve formatting. It was leftover from having the Founding members title as well.